x/oracle/keeper/oracle.go

```diff
@@ -134,6 +178,27 @@ func (k Keeper) SetOracleRegistration(ctx sdk.Context, regOracle *types.OracleRe
 	return nil
 }
 
+func (k Keeper) GetAllOracleRegistrationVote(ctx sdk.Context) ([]types.OracleRegistrationVote, error) {
+	store := ctx.KVStore(k.storeKey)
+	iterator := sdk.KVStorePrefixIterator(store, types.KeyPrefixOracleRegistrationVote)
+	defer iterator.Close()
+
+	oracleRegistrationVotes := make([]types.OracleRegistrationVote, 0)
+
+	for ; iterator.Valid(); iterator.Next() {
+		bz := iterator.Value()
+		var oracleRegistrationVote types.OracleRegistrationVote
+		err := k.cdc.UnmarshalLengthPrefixed(bz, &oracleRegistrationVote)
+		if err != nil {
+			return nil, sdkerrors.Wrapf(types.ErrGetOracleRegistration, err.Error())
```

Comment:
When error occurs, an error like `ErrGetOracleRegistrationVote` seems to be returned, not `ErrGetOracleRegistration`.